app/assets/javascripts/components/header/header.es6.jsx

```diff
@@ -64,14 +64,18 @@ class Header extends Component {
   // --------------------------------------------------
   componentWillMount() {
     this.setState(HeaderStore.getState());
+    this.setState(ProfileStore.getState());
```

Comment:
Because the `Header` is always rendered by a `...Page` component, couldn't we just pass the `Header` a `profile` prop from every page and reduce the number of `ProfileActions.fetchProfile();`'s?